platform/android/os_android.cpp

```diff
@@ -400,33 +400,33 @@ void OS_Android::process_touch(int p_what, int p_pointer, const Vector<TouchPos>
 				input->parse_input_event(ev);
 			}
 
-			ERR_FAIL_COND(touch.size() != p_points.size());
-
-			for (int i = 0; i < touch.size(); i++) {
+			//ERR_FAIL_COND(touch.size() != p_points.size());
```

Comment:
Why remove the error? If you comment it out, you should give a reason why it's better to keep it commented than to remove it. If it's not necessary anymore, just remove it - but here you don't use the `else` branch, so I'm not sure why it should no longer be an error.

Reply:
Nevermind, just read the comment in your PR. If it was written as an error and you are getting spammed with that error, that might hint at a bug though, which should not be hidden unless you're sure that `touch.size() != p_points.size()` is a valid use case.

Reply:
CC @RandomShaper

Reply:
I think touch.size() will always be 0 since there is no touch on a hover event. p.point should contain the previous hover positions. see https://developer.android.com/reference/android/view/MotionEvent#ACTION_HOVER_MOVE

Reply:
I think a hover should be understood by Godot as an update in the mouse position.
Anyway, some stuff about input handling changes in #17196 and I'd strongly advise to base any work of this kind upon it.